src/WeatherLibrary/WeatherEnum.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace WeatherLibrary;
+
+public class WeatherEnum
+{
+    public enum Language
```

Comment:
```suggestion
public enum Language
```

enum не обязательно должен быть внутри класса
enum это самостоятельный тип, поэтому достаточно просто его описать в файле